### PR TITLE
some cleanup

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

**Comment by j0k3r** [comment]
No no no no, don't remove `.editorconfig` :exclamation::exclamation::exclamation: :-1: 

:arrow_right:  http://editorconfig.org/

> EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs.

And install this https://github.com/sindresorhus/editorconfig-sublime or this https://github.com/sindresorhus/atom-editorconfig

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,10 @@
             "name": "Thomas Citharel",
             "homepage": "http://tcit.fr",
             "role": "Developer"
+        },
+        {
+            "name": "Jérémy Besnoit",
```
Comment (j0k3r) [inline composer.json]: And stop calling me _Besnoit_ when it's *Benoist* :)

_Yes you can also update it in your email client_ :stuck_out_tongue:

Comment (nicosomb) [inline composer.json]: Oh sorry for that!

**Comment by j0k3r** [comment]
All right, you can squash your commits now :)

**Comment by nicosomb** [comment]
you can merge now ;-)

**Pull request merged**